docs/application/web/guides/w3c/perf-opt/performance-improvement.md

```diff
@@ -286,6 +286,18 @@ The basic principle of improving the launch time of a Web application is simply
   
   Often the first page can contain unnecessary elements, and consequently slow down the DOM construction. To avoid the problem, construct the first page to contain only the necessary elements to show the first scene, and put the rest of the pages in another HTML file. Similarly, if you do not need some JavaScript functionality on the first page, load the related JavaScript files only after the first page is loaded.
 
+- Use web frameworks appropriately.
+
+  There are many popular web frameworks and they help us create web applications easier. It is possible to use popular web frameworks such as Angular, Vue and so on for Tizen web application development.
```

Comment:
Hello, @Reni-90 and @msaltnet 
Thanks for the review and update. I checked the comment history and think the discussion was good to apply.
However when I read the review result, I got some minor requirements to update the sentences.
Would you please check my below suggestion and apply it if they are acceptable?

---

There are many popular web frameworks and they help us create web applications easier. It is possible to use popular web frameworks such as Angular, Vue and so on for Tizen web application development. 

However, if you do not use a framework correctly, it can degrade your application performance on the contrary.

For example, application launch can take a long time. The delay is because most of the web frameworks must be initialized for resource loading while the application starts.
So it is recommended not to use web frameworks if your web application UX is simple and the launch time performance is important.

Alternatively, avoid loading web frameworks until the web application displays the first page. You can create a simple first page that does not use web frameworks and load web frameworks asynchronously while the first page is displayed. Additionally most frameworks have specific features to improve their loading performance, for example, lazy-loading. To improve the performance of your application, you can use one of that specific features in your web applications that are based on web frameworks.


@Reni-90 Thanks for the review again. I hope you also recheck the overall flow of the sentences in this "Use web frameworks appropriately" section. If you get any point to fix, please leave comments.

Reply:
@mijin85cho 
The suggestion looks good.
Only minor points.
1. "So it is recommended not to use web frameworks if your web application UX is simple and the launch time performance is important."

Here, we can change "So" to "Therefore,". This is just to highlight the recommendation.

2.  You can create a simple first page that does not use web frameworks and load web frameworks asynchronously while the first page is displayed. Additionally most frameworks have specific features to improve their loading performance, for example, lazy-loading. To improve the performance of your application, you can use one of that specific features in your web applications that are based on web frameworks.

Here, a request to add a comma (,) after "Additionally". In "...one of that specific features...", change the determiner in "that specific feature". We can rephrase as "...one of the specific features...".
Please verify.
@msaltnet 
Please verify and update.

Reply:
Thanks. I will update.